web/forms.py

```
@@ -1787,3 +1792,54 @@ class StudyGroupForm(forms.ModelForm):
     class Meta:
         model = StudyGroup
         fields = ["name", "description", "course", "max_members", "is_private"]
+
+
+class MembershipPlanForm(forms.ModelForm):
+    def clean_stripe_monthly_price_id(self) -> str:
+        price_id = self.cleaned_data.get("stripe_monthly_price_id")
+        if price_id and not price_id.startswith("price_"):
+            raise forms.ValidationError("Invalid format: Stripe price IDs must start with 'price_'")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)